### ISSUE TITLE
Can't open links in Skype

### ISSUE THREAD

**Issue by kytwb** [getferdi/ferdi#36]
https://github.com/meetfranz/franz/issues/1167#issuecomment-531567535

> Neither are in my Franz. I click skype links and they won't open up, unless i copy and paste them outside. Before an update, i could click them, only to find out an error page. Maybe Microsoft doesn't quite like to be filtered out like that.

**Comment by bittner** [comment]
We should make sure to make fixes to issues like that in upstream. So we don't need to manage the code delta as an overhead here.

**Comment by vantezzen** [comment]
I've looked into this and the Skype client uses the following (unusual) technique to open links:
```JavaScript
// Simplified version of Skype's t.prototype._openUrl function
function openURL(url) {
    // Open a blank new window first
    const newWindow = window.open();

    if (newWindow) {
        // Check that its not a mailto link
        if (url.indexOf("mailto:") !== 0) {
            // Let the new window open the URL
            newWindow.location.href = url
        }
    }
}
```

Franz, however, expects the service to open the URL directly, like
```
window.open(url);
```

@kytwb it looks like you removed yourself from this issue? I would create a fix for the Franz repo in this case.

**Comment by vantezzen** [comment]
Submitted fix in [franz#1621](https://github.com/meetfranz/franz/pull/1621)

**Comment by kytwb** [comment]
@vantezzen well done! 💪

**Comment by bittner** [comment]
Wow, that's actually awesome! An issue that is bugging users for months, solved via the renegade spin-off project. :100: :crossed_swords: 

BTW, we may call the dark-blue label "contribute upstream". (I know, this is maybe less self-explanatory for everyone, unlike "Implement into Franz", but I think this is somewhat the official term of fixing stuff in the parent project.)

**Comment by kytwb** [comment]
Can we implement the fix into Ferdi already? As it seems to take some time to get implemented in upstream.

**Comment by vantezzen** [comment]
Yes, the fix is now implemented into Ferdi as of 438b9fe.

**Comment by kytwb** [comment]
Shipped in [v5.3.4-beta.2](https://github.com/getferdi/ferdi/releases/tag/v5.3.4-beta.2)
